src/Microsoft.VisualStudio.Editor.Razor/BlockKind.cs

```diff
@@ -19,6 +19,8 @@ public enum BlockKind
 
         // Special
         Comment,
-        Tag
+        Tag,
+
+        HtmlComment
```

Comment:
What kind of reaction is needed from WTE for this?

How much testing of this change have we done in the IDE?

Reply:
I've talked to @ToddGrun about this, and asked him check whether this will impact them or not. He said he'll confirm it today.